fly/integration/logout_test.go

```diff
@@ -49,180 +79,64 @@ var _ = Describe("logout Command", func() {
 	})
 
 	Describe("delete all", func() {
-		var (
-			flyrc  string
-			tmpDir string
-		)
-
-		BeforeEach(func() {
-			var err error
-			tmpDir, err = ioutil.TempDir("", "fly-test")
-			Expect(err).NotTo(HaveOccurred())
-
-			if runtime.GOOS == "windows" {
-				os.Setenv("USERPROFILE", tmpDir)
-				os.Setenv("HOMEPATH", strings.TrimPrefix(tmpDir, os.Getenv("HOMEDRIVE")))
-			} else {
```

Comment:
Actually there seems to be several places in the fly unit and integration tests where it checks the `GOOS`, not sure what their purposes are

Reply:
We run the `fly` tests across all 3 platforms in the pipeline

Reply:
Yeah, but not on PRs. Have tested this on Windows, though. FYI it works without setting `USERPROFILE` and `HOMEPATH` because we always check for `HOME` first, and fallback to `USERPROFILE` and `HOMEPATH` on windows:

https://github.com/concourse/concourse/blob/38e7816949904367a7eb51ebf2ceafad3a401ba3/fly/rc/targets.go#L156-L175